CHANGES.md

```diff
@@ -1,12 +1,13 @@
 # legacysurvey Change Log
 
-## 9.2.0 (DR9, unreleased)
-
-- *Planned*: correct units for `ccdskycounts`
```

Comment:
We should leave these notes as "planned" until they are actually implemented, rather than deleting them entirely.

Reply:
OK, I added:

Planned: correct units for ccdskycounts (#152).
Planned: actually update the raw and CP-processed data access instructions (#143).

back into the changes docs.

Reply:
I _didn't_ retain the point about a gallery being planned for DR9, as I think we've moved past that issue.

Reply:
Yes, that's fair.